apps/ledger-live-mobile/src/locales/en/common.json

```diff
@@ -492,7 +492,7 @@
       "description": "Try reinstalling the app from My Ledger"
     },
     "NotEnoughGas": {
-      "title": "You need {{fees}} {{ticker}} for network fees to swap as you are on {{cryptoName}} network. <link0>Buy {{ticker}}</link0>",
+      "title": "You need {{fees}} {{ticker}} for network fees to send as you are on {{cryptoName}} network. <link0>Buy {{ticker}}</link0>",
```

Comment:
I noticed this on LLD so i guess we need to also update it ?

Reply:
Maybe just be a bit more generic eg. `You need {{fees}} {{ticker}} for fees as you are on {{cryptoName}} network`

Reply:
Fixed